base_multi_company/hooks.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2015-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import SUPERUSER_ID, api
+
+__all__ = [
+    "post_init_hook",
+    "uninstall_hook",
+]
+
+
+def set_security_rule(env, rule_ref):
+    """Set the condition for multi-company in the security rule.
+
+    :param: env: Environment
+    :param: rule_ref: XML-ID of the security rule to change.
+    """
+    rule = env.ref(rule_ref)
+    if not rule:  # safeguard if it's deleted
+        return
+    rule.write(
+        {
+            "active": True,
+            "domain_force": (
+                "['|', ('company_ids', '=', False), ('company_ids', "
```

Comment:
@LoisRForgeFlow Have you ever experienced issues with `company_ids = False `domain ?

I fixed it with `company_ids = []`

Reply:
> @LoisRForgeFlow Have you ever experienced issues with `company_ids = False `domain ?
> 
> I fixed it with `company_ids = []`

Moreover, 'in' [] is the correct one.